ouroboros-network-framework/src/Ouroboros/Network/Protocol/Handshake/Version.hs

```diff
@@ -85,6 +87,11 @@ data DictVersion vData where
                  => CodecCBORTerm Text vData
                  -> DictVersion vData
 
+data CodecCBORTerm fail a = CodecCBORTerm
```

Comment:
I agree; since we only use it for the `Handshake` protocol it's better to dock it in here.